docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -356,13 +426,15 @@ Standard Kaldi\* Layers:
 | :----------| :----------|
 | addshift | No |
 | affinecomponent | No |
+| affinecomponentpreconditionedonline | No |
 | affinetransform | No |
+| backproptruncationcomponent | No |
+| batchnormcomponent | No |
 | clipgradientcomponent | Not needed for inference |
 | concat | No |
 | convolutional1dcomponent | No |
 | convolutionalcomponent | No |
 | copy | No |
-| Crop | No |
```

Comment:
We have an extractor for Kaffe Crop operation: model-optimizer/mo/front/kaldi/extractors/crop_ext.py

Reply:
Yes, but @sadolini asked to remove. @sadolini do we support Crop in Kaldi?

Reply:
I discussed it offline with @sadolini. Kaldi framework doesn't have "Crop" layer. In MO Kaldi loader replaces construction dim-range-node with 'Crop' operation and we use this extractor for this internal 'Crop' operation.

Reply:
Ok. Thanks for the explanation.